examples/gnrc_tcp_srv/Makefile

```diff
@@ -0,0 +1,61 @@
+# name of your application
+APPLICATION = gnrc_tcp_srv
+
+CC = clang
+LD = clang
+
+# If no BOARD is found in the environment, use this default:
+#BOARD ?= native
+BOARD ?= samr21-xpro
+#PORT = tap1
+#PORT = /dev/ttyACM0
+
+SERIAL = "ATML2127031800002301"
+
```

Comment:
These defines are in there to simplyfiy my personal testing. They will removed during the final cleanup ;)